modules/oe_webtools_analytics/src/AnalyticsEventInterface.php

```diff
@@ -56,13 +56,21 @@ interface AnalyticsEventInterface extends \JsonSerializable, RefinableCacheableD
    * Refine the statistics by indicating a site section  or a subwebsite.
    */
   public const SITE_SECTION = 'siteSection';
+
   /**
    * Set to true when search with the parameters form SearchParameters class.
    *
    * @see \Drupal\oe_webtools_analytics\Search\SearchParametersInterface
    */
   public const SEARCH = 'search';
 
+  /**
+   * Set to true when search with the parameters form SearchParameters class.
```

Comment:
This comment doesn't seem correct. Maybe: "Allows to define additional custom dimensions." (taken from the webtools docs)